src/components/EmailButton.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import PropTypes from "prop-types";
+import contact from "../data/contact.json";
+
+function EmailButton({ mode }) {
+  return (
+    <a href={`mailto:${contact.email}?subject=${contact.emailSubject} ${mode}`}>
+      <button className="btn blue-btn">Anmelden</button>
+    </a>
+  );
+}
+
+EmailButton.propTypes = {
```

Comment:
@angsherpa232 you canot know this but we usually define `propTypes` and `defaultProps` at the top of the file.
See e.g. https://github.com/geops/react-spatial/blob/master/src/components/Dialog/Dialog.js